src/core/continuous_space.jl

```diff
@@ -146,3 +149,35 @@ function move_agent!(agent::A, model::ABM{A, S, F, P}) where {A<:AbstractAgent,
   end
   return agent.pos
 end
+
+function kill_agent!(agent::AbstractAgent, model::ABM{A, S}) where {A, S<:ContinuousSpace}
+  DBInterface.execute(model.space.deleteq, (agent.id,))
+  delete!(model.agents, agent.id)
+  return model
+end
+
+function genocide!(agentIDs::AbstractArray{Int}, model::ABM{A, S}) where {A, S<:ContinuousSpace}
+  DBInterface.execute(model.space.db,
+  "DELETE FROM tab WHERE id IN $(Tuple(agentIDs))")
+  for id in agentIDs
+    delete!(model.agents, id)
+  end
+  return model
+end
+
+function genocide!(agents::AbstractArray, model::ABM{A, S}) where {A, S<:ContinuousSpace}
+  DBInterface.execute(model.space.db,
+  "DELETE FROM tab WHERE id IN $(Tuple([a.id for a in agents]))"

```

Comment:
Once again here `[]` is not necessary. `Tuple(rand() for i in 1:100)` is valid syntax. But making large tuples is bad for performance. What is the goal here? Is it to just interpolate into a string all ids? `"1, 23, 5, 6"` with commas?

Reply:
we need `(1,2,3...)` in the string.

Reply:
Then please use the same approach I have done for the other queries using `join(...)`. String operation and manipulation is what we need at this occassion, and will also bring performance benefits.